pkg/agent/openflow/client_windows.go

```diff
@@ -51,3 +54,33 @@ func (c *client) UninstallLoadBalancerServiceFromOutsideFlows(svcIP net.IP, svcP
 	cacheKey := fmt.Sprintf("L%s%s%x", svcIP, protocol, svcPort)
 	return c.deleteFlows(c.serviceFlowCache, cacheKey)
 }
+
+func (c *client) InstallServiceFlows(groupID binding.GroupIDType, svcIP net.IP, svcPort uint16, protocol binding.Protocol, affinityTimeout uint16) error {
+	c.replayMutex.RLock()
+	defer c.replayMutex.RUnlock()
+	var flows []binding.Flow
+	flows = append(flows, c.serviceLBFlow(groupID, svcIP, svcPort, protocol, affinityTimeout != 0))
+	if affinityTimeout != 0 {
+		flows = append(flows, c.serviceLearnFlow(groupID, svcIP, svcPort, protocol, affinityTimeout))
+	}
+	if c.enableProxy {
+		// ClusterIP Service requests will be sent to OVS bridge via ServiceGWVirtuallP. This NetNeighbor is for
+		// creating a neighbor cache entry to ServiceGWVirtuallP.
+		util.NewNetNeighbor(c.nodeConfig.GatewayConfig.LinkIndex, route.ServiceGWVirtuallP, route.VirtualSvcMAC)
```

Comment:
I think funcs in openflow client should just handles OF flows, but not other stuff like neighbor entries. Could we add a funcs in route client to add/remove neighbor entries, which should be called by proxier directly.

Reply:
Moved to route client.